module/EtRobocon2023.cpp

```diff
@@ -35,4 +37,52 @@ void EtRobocon2023::start()
   Measurer::rightMotor = &_rightMotor;
   Measurer::leftMotor = &_leftMotor;
   Measurer::armMotor = &_armMotor;
+
+  const int BUF_SIZE = 128;
+  char buf[BUF_SIZE];  // log用にメッセージを一時保持する領域
+  Logger logger;
+
+  bool isLeftCourse = true;
+  bool isLeftEdge = true;
+  int targetBrightness = (WHITE_BRIGHTNESS + BLACK_BRIGHTNESS) / 2;
+  Calibrator calibrator;
+
+  // 強制終了(CTRL+C)のシグナルを登録する
+  signal(SIGINT, sigint);
+
+  // キャリブレーションする
+  calibrator.run();
+  isLeftCourse = calibrator.getIsLeftCourse();
+  isLeftEdge = isLeftCourse;
+  targetBrightness = calibrator.getTargetBrightness();
+
+  // 合図を送るまで待機する
+  calibrator.waitForStart();
+
+  // スタートのメッセージログを出す
+  const char* course = isLeftCourse ? "Left" : "Right";
+  snprintf(buf, BUF_SIZE, "\nRun on the %s Course\n", course);
+  logger.logHighlight(buf);
+
+  // 各エリアを走行する
+  AreaMaster lineTraceAreaMaster(LineTrace, isLeftCourse, isLeftEdge, targetBrightness);
+  AreaMaster doubleLoopAreaMaster(DoubleLoop, isLeftCourse, isLeftEdge, targetBrightness);
+  AreaMaster blockDeTreasureAreaMaster(BlockDeTreasure, isLeftCourse, isLeftEdge, targetBrightness);
```

Comment:
ついてた方がいいかな。
[記事](https://monozukuri-c.com/langcpp-funclist-enum-class/#toc-3)によっては型名(Area)の記述が必須らしい

```suggestion
  AreaMaster lineTraceAreaMaster(Area::LineTrace, isLeftCourse, isLeftEdge, targetBrightness);
  AreaMaster doubleLoopAreaMaster(Area::DoubleLoop, isLeftCourse, isLeftEdge, targetBrightness);
  AreaMaster blockDeTreasureAreaMaster(Area::BlockDeTreasure, isLeftCourse, isLeftEdge, targetBrightness);
```